_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,51 @@
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
+
+OmbuLabs has developed a Snap Mini and in this article I'll share with you the main advantages of having a Mini and how we can help you get in on this.
+
+<!--more-->
+
+First of all, why create a Snap Mini? Here are the main advantages of having your own app inside [Snapchat](https://www.snapchat.com/):
+
+## Easy to Develop
+Snap Minis exist inside the main Snapchat application. They are built on HTML5 and take advantage of the [Snap Kit](https://kit.snapchat.com/), reducing the scope of the project.
+
+Additionally, Minis work on both Android and iOS devices. You do not need two different applications to cater to different users.
+
+## Easy to Use
+The Minis are lightweight applications easily accessible through the Chat interface of the Snapchat app. Users do not need to install a separate app and switch applications to use your service. They also don't need to wait for a download to finish.
+
+From inside the Snapchat application right in the conversation your users are having with their friends, they can pull up the Games menu and your Mini is readily available for them to use.
+
+## Easy to Share with Friends
+A Mini can interact with both the chat and camera interfaces of the Snapchat application. This makes it easy for your users to share that they've used your mini with their friends.
+
+Their friends can then tap the chat card or camera sticker and be taken to your mini. All without leaving Snapchat.
+
+# What Can I do with Minis?
+
+There are several possibilities here. Minis can integrate with the Snapchat features and perform a variety of actions.
+
+Having in mind the concept of a Mini, a bite-sized application inside the Chat interface, you can define an actionable use case within your product and turn it into a mini. Some examples shared by Snap during the Snap Partner Summit 2020 include deciding what to watch, buying tickets, voter registration and health and fitness. If you have a specific action that can be performed by your audience, you can turn it into a Mini.
+
+Another advantage is adding a social aspect to the experience. These mini applications live inside the Chat interface and allow friends to browse and use them together. You can take advantage of the communication that's already happening on Snapchat and bring that into your service.
```

Comment:
@ABizzinotto I think it would be interesting to see the social aspect of the mini and how it looks. I think it is pretty awesome that you can see which friends are online and playing the mini app. 

Could you add a snapshot that shows more than one people in the footer of a mini?

Reply:
@etagwerker I don't think adding a snapshot here is a good idea. Let's Do It is only out for New Zealand and the other Minis aren't even out yet. 

I can add a line like:

"A pretty awesome feature of minis is you can see in real time which of your friends are online and on that same mini at the same time as you".

What do you think?

Reply:
@ABizzinotto sounds good.